[Bug] Grafana Test Data Divide by Zero exception

http://docs.grafana.org/features/datasources/testdata

If you change the data source for the CSV from the default "1,20,90,30,5,0" to "50" the server crashes with DivideByZeroException.

Grafana command line output:

INFO[06-26|09:22:02] Initializing HTTP Server                 logger=http.server address=0.0.0.0:3000 protocol=http subUrl= socket=
panic: runtime error: integer divide by zero

goroutine 80 [running]:
github.com/grafana/grafana/pkg/tsdb/testdata.init.1.func4(0xc0426b2700, 0xc04266d620, 0xc04266b920)
        c:/gopath/src/github.com/grafana/grafana/pkg/tsdb/testdata/scenarios.go:111 +0x7d6
github.com/grafana/grafana/pkg/tsdb/testdata.(*TestDataExecutor).Execute(0xc0426bc6e0, 0x17fa020, 0xc04203a310, 0xc042216600, 0x1, 0x1, 0xc04266d620, 0x0)
        c:/gopath/src/github.com/grafana/grafana/pkg/tsdb/testdata/testdata.go:34 +0x218
github.com/grafana/grafana/pkg/tsdb.(*Batch).process(0xc0426c0330, 0x17fa020, 0xc04203a310, 0xc04266d620)
        c:/gopath/src/github.com/grafana/grafana/pkg/tsdb/batch.go:39 +0x33d
created by github.com/grafana/grafana/pkg/tsdb.HandleRequest
        c:/gopath/src/github.com/grafana/grafana/pkg/tsdb/request.go:23 +0x227

Message from Grafana in browser window:
Problem!
[Fiddler] The connection to 'localhost' failed. <br />Error: ConnectionRefused (0x274d). <br />System.Net.Sockets.SocketException No connection could be made because the target machine actively refused it 127.0.0.1:3000

--- a/pkg/tsdb/testdatasource/scenarios.go
+++ b/pkg/tsdb/testdatasource/scenarios.go
@@ -237,7 +237,10 @@ func init() {
 			series := newSeriesForQuery(query, 0)
 			startTime := context.TimeRange.GetFromAsMsEpoch()
 			endTime := context.TimeRange.GetToAsMsEpoch()
-			step := (endTime - startTime) / int64(len(values)-1)
+			var step int64 = 0
+			if len(values) > 1 {
+				step = (endTime - startTime) / int64(len(values)-1)
+			}
 
 			for _, val := range values {
 				series.Points = append(series.Points, tsdb.TimePoint{val, null.FloatFrom(float64(startTime))})

